From: Richard B Winters <rik@mmogp.com>
Date: Wed, 6 May 2015 01:37:08 -0400
Subject: Fix pcsd makefile for gnu linux

 - Patching pcsd makefile with upstream's latest changes, so 
   that the build works properly on debian. Without the patch, 
   the incorrect alias is used for the build_gems target

Change-Id: I004a27b8982594a393c72ba3abfa7425a5ec2687
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 pcsd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pcsd/Makefile b/pcsd/Makefile
index 325f54a..8779671 100644
--- a/pcsd/Makefile
+++ b/pcsd/Makefile
@@ -1,9 +1,9 @@
 REL_INFO := $(shell grep -q -i "release 6" /etc/redhat-release ; echo $$?)
 
-ifeq (${REL_INFO},1)
-  build_gems: build_gems_normal
-else
+ifeq (${REL_INFO},0)
   build_gems: build_gems_rhel6
+else
+  build_gems: build_gems_normal
 endif
 
 build_gems_normal:
